subprojects/settings.gradle.kts

```diff
@@ -149,22 +146,21 @@ pluginManagement {
         }
     }
 
-    plugins {
-        id("nebula.integtest") version "7.0.7"
-    }
-
     resolutionStrategy {
         eachPlugin {
             val pluginId = requested.id.id
             when {
                 pluginId.startsWith("com.android.") ->
-                    useModule("com.android.tools.build:gradle:$androidGradlePluginVersion")
+                    useModule("com.android.tools.build:gradle:4.1.1")
 
                 pluginId.startsWith("org.jetbrains.kotlin.") ->
-                    useVersion(kotlinVersion)
+                    useVersion("1.4.21")
```

Comment:
We duplicate versions multiple times. 
Let's read the from properties with `forUseAtConfigurationTime`

Reply:
we duplicate it anyway between `root project`, `subprojects` and `buildscript` it it's gradle,properties

i don't see a problem in it

Reply:
let me try though, but we only go from 3 places to 2 with it

Reply:
can't reuse `forUseAtConfigurationTime` in `Dependencies.kt` which reduces reuse to ..  well... nothing)